### ISSUE TITLE
Nested ExpandableBlocks do not show proper icon

### ISSUE THREAD

**Issue by toddsouthenbentley** [iTwin/iTwinUI#1375]
#### Describe the bug (actual behavior)
ExpandableBlock instances inside ExpandableBlock do not display the correct icon (always shown as expanded)

#### Expected behavior
The icon should change and animate when the inner ExpandableBlock is expanded/collapsed. 

#### Reproduction
Link to a minimal repro: https://codesandbox.io/s/proud-meadow-1bqy7h?file=/src/App.tsx

##### Steps to reproduce
```tsx
export default function App() {
  return (
    <>
      <ExpandableBlock title="One" isExpanded={true}>
        <ExpandableBlock title="Two" isExpanded={true}>
          Inner Expandable Block
        </ExpandableBlock>
      </ExpandableBlock>
    </>
  );
}
```

**Comment by FlyersPh9** [comment]
This is an issue we are aware of from a Microsoft Teams discussion.  We believe we have a solution & it is being incorporated in our next major release.  Will consider back porting once we confirm this indeed does fix the issue.

**Comment by toddsouthenbentley** [comment]
> This is an issue we are aware of from a Microsoft Teams discussion. We believe we have a solution & it is being incorporated in our next major release. Will consider back porting once we confirm this indeed does fix the issue.

@FlyersPh9 Is there possibly a known simple workaround I can apply to my own SCSS I can use in the meantime?

**Comment by FlyersPh9** [comment]
@toddsouthenbentley I've created a PR going into the `main` branch that should solve the issue.  I think the PR should be able to merge fairly soon (within ~24 hours).

**Comment by gretanausedaite** [comment]
This is fixed in iTwinUI-react@2.11.7.
